src/Nethermind/Nethermind.Runner/Ethereum/Steps/InitDatabase.cs

```diff
@@ -64,11 +68,11 @@ public async Task Execute()
                 _context.NodeDataConsumer = beamSyncProvider.NodeDataConsumer;
             }
 
-            // IDbProvider debugRecorder = new RocksDbProvider(Path.Combine(_context._initConfig.BaseDbPath, "debug"), dbConfig, _context._logManager, _context._initConfig.StoreTraces, _context._initConfig.StoreReceipts);
-            // _context._dbProvider = new RpcDbProvider(_context._jsonSerializer, new BasicJsonRpcClient(KnownRpcUris.Localhost, _context._jsonSerializer, _context._logManager), _context._logManager, debugRecorder);
-
-            // IDbProvider debugReader = new ReadOnlyDbProvider(new RocksDbProvider(Path.Combine(_context._initConfig.BaseDbPath, "debug"), dbConfig, _context._logManager, _context._initConfig.StoreTraces, _context._initConfig.StoreReceipts), false);
-            // _context._dbProvider = debugReader;
+            // RocksDbProvider debugRecorder = new RocksDbProvider(_context.LogManager);
```

Comment:
is there a way to do that with some config switch so we do not have to comment out / comment  back in?

Reply:
Added diagnostic enumeration property